src/omv/modules/py_spi_display.c

```diff
@@ -394,6 +414,7 @@ STATIC const py_display_p_t py_display_p = {
     #ifdef OMV_SPI_DISPLAY_BL_PIN
     .set_backlight = spi_display_set_backlight,
     #endif
+    .bus_write = spi_write,
```

Comment:
No `bus_read` ?

Reply:
We didn't connect it on the LCD shield so I don't have anything to test with at the moment. I can write a new display controller class for like a 320x240 screen and then test and add read support later.